### PR TITLE
Set hit on /r research.

### PR DESCRIPTION
**Pull request by gogo2464**
- [x] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

**Description**

The command `/` already sets a flag contrary to `/r` then I put a flag on hits for `/r`. 

I am waiting for opinion as well as review.

### CODE REVIEW

File: libr/core/cmd_search.c [hunk 1]
```diff
@@ -3985,7 +3988,7 @@ static int cmd_search(void *data, const char *input) {
 					}
 					if (r_cons_is_breaked ()) {
 						break;
-					}
+					}//r_flag_set (core->flags, flag, base_addr + addr, klen);
```
Comment (gogo2464) [inline libr/core/cmd_search.c:3991]: my bad

File: libr/core/cmd_search.c [hunk 2]
```diff
@@ -2192,6 +2192,9 @@ static void do_ref_search(RCore *core, ut64 addr,ut64 from, ut64 to, struct sear
 			if (from <= ref->addr && to >= ref->addr) {
 				r_cons_printf ("%s 0x%" PFMT64x " [%s] %s\n",
 						buf_fcn, ref->addr, r_anal_ref_type_tostring (ref->type), str);
+
+				r_flag_set (core->flags, "hit.prelude", ref->addr, strlen ("hit.prelude"));
```
Comment (trufae) [inline libr/core/cmd_search.c:2196]: there are several wrong things in this line:

* why hit.prelude? /r have nothing to do with preludes, its about references
* this codepath is executed many times, one for each reference found, your pr is overwriting the same flag over and over
* why the flag size is the length of a string? thats unrelated to the reference it points to
* xrefs are stored as xrefs, thats why no flags are set when using /r

Comment (trufae) [reply libr/core/cmd_search.c:2196]: the 'r' in /r stands for reference, not for research

**Comment by gogo2464** [comment]
`xa` does what I want instead of `f`. Could you close this PR please?

Sorry for the inconvenience.

**Pull request closed without merging**